cmd/kubeadm/app/phases/kubelet/flags_test.go

```diff
@@ -142,7 +142,7 @@ func TestBuildKubeletArgMap(t *testing.T) {
 			},
 		},
 		{
-			name: "dockershim socket but kubelet version is without built-in dockershim",
+			name: "docker socket but kubelet version is without built-in dockershim",
```

Comment:
Does the dockershim socket(`/var/run/dockershim.sock`) is same as the docker socket(`/var/run/cri-dockerd.sock`) here?

Reply:
I prefer cri-dockerd

Reply:
actually there is no need to change the "name" field here. i will revert this.
the tests are going to removed in 1.25.